### PR TITLE
Fixed reference order and added two new references

### PR DESCRIPTION
**Pull request by sichaox**

### CODE REVIEW

File: references.bib [hunk 1]
```diff
@@ -911,5 +911,23 @@ @Article{Stuke2020
...
+}
+
+@misc{CCCBDB,
+  author = {Russell Johnson},
+  title = {Sicklist Database},
+  year = {1999},
+  month = {1999-10-01},
+  publisher = {CCCBDB Computational Chemistry Comparison and Benchmark Database Sicklist},
+  language = {en},
+}
```
Comment (ndattani) [inline references.bib]: I don't think the [sicklist](https://www.nist.gov/publications/sicklist-database) is the correct reference.

File: PossibleGroupsForMolecules.lyx [hunk 2]
```diff
@@ -4,7 +4,7 @@
 \begin_header
 \save_transient_properties true
 \origin unavailable
-\textclass revtex4-1
+\textclass revtex4-2
```
Comment (ndattani) [inline PossibleGroupsForMolecules.lyx]: Please revert this to 4-1 for now.

File: PossibleGroupsForMolecules.lyx [hunk 3]
```diff
@@ -9035,7 +9032,7 @@ CHBrCl
 LatexCommand bibtex
 btprint "btPrintCited"
 bibfiles "references"
-options "apsrev4-1"
+options "apsrev4-2"
```
Comment (ndattani) [inline PossibleGroupsForMolecules.lyx]: Please revert this to 4-1 for now.

File: references.bib (hunk 1 above)
Comment (sichaox) [reply references.bib]: Sorry I found this
NIST Computational Chemistry Comparison and Benchmark Database,
NIST Standard Reference Database Number 101
Release 22, May 2022, Editor: Russell D. Johnson III
http://cccbdb.nist.gov/
However not sure how it translates into a bibtex references.

Comment (sichaox) [reply references.bib]: I accidently looked at the next CCCDBD link and downloaded the sicklist. 😂

Comment (ndattani) [reply references.bib]: A bibtex entry is very likely available somewhere, and otherwise you can make it yourself.

**Comment by ndattani** [comment]
I did not ask you to close this pull request, just to revert the last commit.

**Comment by ndattani** [comment]
@sichaox why have you removed all of your commits?

**Comment by ndattani** [comment]
@sichaox why have you removed all of your commits?

**Pull request closed without merging**